source/_components/remote.lg_netcast.markdown

```diff
@@ -0,0 +1,154 @@
+---
+layout: page
+title: "LG Netcast Remote"
+description: "Instructions on how to integrate LG Netcast (Netcast 3.0 & 4.0) remotes into Home Assistant."
+date: 2016-11-05 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lg.png
+ha_category: Remote
+ha_iot_class: "Local Push"
+ha_release: "0.82.0"
+---
+
+The `lg_netcast` remote platform allows you to control the state of your LG Netcast TV, NetCast 3.0 (LG Smart TV models released in 2012) and NetCast 4.0 (LG Smart TV models released in 2013).
+
+
+```yaml
+# Example configuration.yaml entry
+remote:
+  - platform: lg_netcast
+    name: tv_room
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.